File: fibonnaci.php
<?php

function fibonnaci($first, $second, $limit) {
    if ($first === 0) {
//        print 0 . PHP_EOL . 1 . PHP_EOL;
        $newFirst = 1;
        $newSecond = 1;
    } else {
//        print $first . PHP_EOL;
//        print ($first + $second) . PHP_EOL;
        $newFirst = $second;
        $newSecond = $first + $second;
    }

    if ($newSecond < $limit) {
        print $newFirst . ' ' . $newSecond . PHP_EOL;
        fibonnaci($newFirst, $newSecond, $limit);
    }
}

fibonnaci(0, 1, 3);

File: fibonnaci.py
"""
Module that will calculate the fibonnaci sequence
"""

def fibonnaci(first, second, limit):
    if first == 0:
        print 0
        print 1
        new_first = 1
        new_second = 1
    else:
        new_second = first + second
        new_first = second

    if new_second < limit:
        print new_second
        fibonnaci(new_first, new_second, limit)

fibonnaci(0, 1, 35)